coursier.bzl

```diff
@@ -132,7 +122,9 @@ def _execute(repository_ctx, cmd, timeout = 600, environment = {}, progress_mess
 # The representation of a Windows path when read from the parsed Coursier JSON
 # is delimited by 4 back slashes. Replace them with 1 forward slash.
 def _normalize_to_unix_path(path):
-    return path.replace("\\\\", "/")
+    if path == None:
```

Comment:
Why this change? Seems error prone to have `None` paths being passed around.

Reply:
Just being paranoid. I'll back it out.